### ISSUE TITLE
Add grabber for scrolling quickly

### ISSUE THREAD

**Issue by dmazurek** [jpgilchrist/gpscy-karaoke#5]
The list is so big that the scrollbar dingus on the right is about 2-3 pixels tall. You can't grab it. Is there a plugin or something that gives you a big scrollbar for a large list?

**Comment by jermahs** [comment]
Maybe? I can look into it? I didn't feel like getting into paging... so that might be a decent alternative.

**Comment by dmazurek** [comment]
Yeah, I wouldn't bother with paging... it's kinda nice being able to scroll the whole list. There's just no easy way to jump way down -- you gotta keep swiping. :)

**Comment by jermahs** [comment]
Here's this: http://jsfiddle.net/jeremy/Eh7xT/342/ might help!
